.travis.yml

```diff
@@ -47,7 +48,7 @@ script:
   - ./bessctl/bessctl -- daemon start -- run testing/run_module_tests
 
 after_success:
-  - "[[ ${DEBUG:-0} == 0 ]] || bash <(curl -s https://codecov.io/bash)"
+  - "[[ ${DEBUG:-0} == 0 ]] || (bash <(curl -s https://codecov.io/bash) && cd core && ./coverage run)"
```

Comment:
How about `(( ... || ...) && ( ... && ...))`, to make the command side-effect-free and clearer in terms of operator precedences?